pkgs/misc/emulators/citra/base.nix

```
@@ -0,0 +1,73 @@
+{ pname, version, src, branchDesc, branchName
+, lib, stdenv, config, fetchFromGitHub
+, catch2, cmake, ninja, pkg-config, wrapQtAppsHook
+, boost173, ffmpeg, libusb1, qtbase, qtdeclarative, qtmultimedia, qttools, rapidjson, SDL2
+, alsaSupport      ? stdenv.isLinux,                      alsa-lib
+, discordSupport   ? true
+, fdkSupport       ? true,                                fdk_aac
+, jackaudioSupport ? true,                                libjack2
+, onlineSupport    ? true
+, pulseSupport     ? config.pulseaudio or stdenv.isLinux, libpulseaudio
```

```suggestion
, pulseSupport     ? stdenv.isLinux, libpulseaudio
```
this causes unnecessary rebuilds when using pipewire